packages/solana/src/utils/scaffold/SolanaHelpersUtils.ts

```diff
@@ -7,7 +8,7 @@ import type { Chain, Provider } from './SolanaTypesUtil.js'
 
 export const SolHelpersUtil = {
   detectRpcUrl(chain: Chain, projectId: string) {
-    if (chain.rpcUrl.includes('rpc.walletconnect.com/')) {
+    if (chain.rpcUrl.includes(ConstantsUtil.BLOCKCHAIN_API_RPC_URL)) {
```

Comment:
@tomiir The previous version doesn't include the protocol in the condition;

```js
    if (chain.rpcUrl.includes('rpc.walletconnect.com/')) {
```

While this one have full url as param of `includes`. Maybe this might affect the result? Worth double checking